src/main/java/frc/team2412/robot/subsystem/TargetLocalizer.java

```diff
@@ -99,30 +112,48 @@ public double getVisionYaw() {
     /**
      * Returns the yaw (horizontal angle) to the target position.
      *
-     * @return The yaw to the hub plus turret angle bias (0 is straight ahead,
-     *         positive is clockwise,
+     * @return The yaw to the hub plus turret angle bias (0 is straight ahead, positive is clockwise,
      *         units are degrees).
      */
     public double getTargetYaw() {
         // return 0;
         return getVisionYaw() + shooterSubsystem.getTurretAngleBias();
     }
 
+    /**
+     * Returns the gyroscope's yaw.
+     *
+     * @return The gyroscope's yaw without the adjustment angle (positive rotation is counterclockwise).
```

Comment:
Yes, the reason `getVisionYaw()`, `getTargetYaw()`, and `getTurretAngle()` have positive rotation as clockwise is to avoid interfering with other code (Especially feedforward math, though shooter and shooter vision would also be affected).